src/main/java/io/r2dbc/postgresql/codec/CircleCodec.java

```diff
@@ -50,6 +51,16 @@ ByteBuf doEncodeBinary(Circle value) {
             .writeDouble(value.getRadius());
     }
 
+    @Override
+    public String encodeToText(Circle value) {
+        return String.format("\"%s\"", value);
```

Comment:
`String.format(…)` is rather costly, I'm going to switch instances of `String.format(…)` to string concatenation. `String.format(…)` is fine for exceptions but on hot code paths it creates notable performance drag.

Reply:
You are right, I pushed https://github.com/pgjdbc/r2dbc-postgresql/pull/403/commits/98c34620171095a8e82a5c846a9236d81ea71470 which replaces `String.format(...)` with string concat in some places.